src/main/java/forum/entity/Thread.java

```diff
@@ -0,0 +1,84 @@
+package forum.entity;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name="thread")
```

Comment:
Same comment here about the `name`

Reply:
Commented on your earlier reply about it